CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(QMidi)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} "C:\\Qt\\5.15.0\\msvc2019_64\\")
+
+find_package(Qt5 COMPONENTS Core)
+
+# Common QMidi source files
+set(SOURCES
```

Comment:
You may as well use aux_source_dir.

Reply:
Did not know this command, thank you for the hint. Included it.